src/Ocelot.Provider.Eureka/EurekaProviderFactory.cs

```diff
@@ -1,23 +1,29 @@
 ﻿using Microsoft.Extensions.DependencyInjection;
-
+using Ocelot.Configuration;
 using Ocelot.ServiceDiscovery;
-
+using Ocelot.ServiceDiscovery.Providers;
 using Steeltoe.Discovery;
+using System;
+
+namespace Ocelot.Provider.Eureka;
 
-namespace Ocelot.Provider.Eureka
+public static class EurekaProviderFactory
 {
-    public static class EurekaProviderFactory
+    /// <summary>
+    /// String constant used for provider type definition.
+    /// </summary>
+    public const string Eureka = "Eureka";
```

Comment:
```suggestion
    public const string Eureka = nameof(Eureka);
```

Reply:
So true! What will I do without your reviews, Ray?! 😄

Reply:
@raman-m Ok

Reply:
**Fixed** in commit https://github.com/ThreeMammals/Ocelot/pull/1670/commits/b3f1d551c150967f9fc2178fd5839fc77c841e05

Reply:
@raman-m Ok, got it... public const string Eureka = nameof(Provider.Eureka.Eureka);